Core/Core/DeveloperMenu/ExperimentalFeaturesViewController.swift

```diff
@@ -30,7 +30,7 @@ public class ExperimentalFeaturesViewController: UITableViewController {
             navigationItem.rightBarButtonItem = barButtonItem
         }
 
-        title = "Experimental Features"
+        title = NSLocalizedString("Experimental Features", bundle: .core, comment: "")
```

Comment:
This screen originally was dev-only, but now is user visible in read only. The above button is not translated, since it is still dev-only.